build_tools/circle/build_wheels_arm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
+sudo apt-get update
+sudo apt-get install python3-virtualenv ccache
+python3 -m virtualenv --system-site-packages --python=python3 testenv
```

Comment:
I don't think we need any system-level python package for this, no? We just need a bare-bone python to launch cibuildwheel itself.

```suggestion
python3 -m virtualenv --python=python3 testenv
```

Reply:
fixed.